.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+
+### Summary & motivation
+
+<!-- Simple summary of what the code does or what you have changed. If this is a visual change, please include a screenshot/GIF. -->
+
+### API review
+
+<!-- Delete this section if this change involves no or negligible API changes. -->
```

Comment:
Nit: "negligible" is a subjective call -- maybe let's just leave it at "no API changes."